cypress/plugins/index.js

```diff
@@ -25,6 +25,10 @@ module.exports = (on, config) => {
     ...process.env,
   };
 
+  const { E2E_FOLDER = 'seededFlows' } = process.env;
```

Comment:
This is where we decide which E2E test folder to use for running Cypress. By default it runs the tests in the `seededFlows` folder.

Also, I'm not sure if `seededFlows` and `nonSeededFlows` are the best names for these folders, Naming is hard. 😎

Reply:
Going to go with `seededFlows` and `nonSeededFlows` for now. We can revisit this if folks really don't like it.